components/match2/wikis/stormgate/match_summary.lua

```diff
@@ -152,14 +153,15 @@ function CustomMatchSummary.Game(options, game)
 				HtmlWidgets.Div{css = {margin = 'auto'}, children = {game.header}},
 				MatchSummaryWidgets.Break{},
 			} or nil,
-			CustomMatchSummary.DispalyHeroes(game.opponents[1], {hasHeroes = options.hasHeroes}),
+			CustomMatchSummary.DispalyHeroes(game.opponents[1], {hasHeroes = options.hasHeroes, date = options.date}),
```

Comment:
Date is optional to be provided, if it wasn't needed before, are you sure it's needed now?

Reply:
yeah i know it is optional
just thought i might as well make it somewhat future proof
if you think we should drop the date here again just lmk and i can do it (probably saturday/subday earliest)